frontend/src/body.html

```diff
@@ -0,0 +1,52 @@
+<!--
+SPDX-FileCopyrightText: 2017-2022 City of Espoo
+
+SPDX-License-Identifier: LGPL-2.1-or-later
+-->
+<div id="app">
+  <noscript>
+    <main style="margin: 0 20px">
+      <p>
+        Sivusto vaatii, että JavaScript on sallittu selaimessasi.
+      </p>
+      <p>
+        Sivusto vaatii, että JavaScript on sallittu selaimessasi.
+      </p>
+      <p>
+        This site requires JavaScript to be enabled in your browser.
+      </p>
+    </main>
+  </noscript>
+</div>
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
+    var app = document.getElementById('app')
+    function hasLoaded() {
+      if (app) {
+        var children = app.childNodes
+        for (var i = 0; i < children.length; i++) {
+          var child = children[i]
+          if (child instanceof HTMLElement && child.nodeName !== 'NOSCRIPT') {
+            return true
+          }
+        }
+      }
```

Comment:
Would this become simpler if the `<noscript>` part was outside the app div? I think it could be outside, because JavaScript is not run anyways when its content is shown.

Reply:
This would conflict with `height: 100%` style defined for the app div, so we'd need some CSS tricks to compensate